.github/workflows/e2e.yaml

```diff
@@ -35,13 +35,19 @@ on:
           - "1.27"
           - "1.28"
         default: "1.28"
-      enable_metrics:
+      preexisiting_cluster:
+        type: string
+      create_cluster:
         type: boolean
-        default: false
+        default: true
+        required: true
```

Comment:
Can we collapse this into one environment variable? When would I have a preexisting cluster but want to create a new one? and vice versa?